examples/vacationsApprovalServerless/processes/requestVacations.yml

```diff
@@ -0,0 +1,85 @@
+type: process
+key: requestVacations
+name: Request Vacations
+description: Start a vactions request
+icon: pen-alt # Search icons at https://fontawesome.com/icons?d=gallery&s=solid&m=free
+summary:
+  title: Vacations request for {{requester.firstName}} {{requester.lastName}}
+  subtitle: From {{startDate}} to {{endDate}}
+mainTable: requests
+startsFromExistingData: false
+publicStart: false
+
+permissions:
+  manager:
+    start: true
+    pause: true
+    archive: true
+    reassign: true
+  
+  employee:
+    start: true
+    pause:
+      own: true
+    archive:
+      own: true
+
+fieldGroups:
+  requestData:
+    rows:
+      - requester
+      - [startDate, endDate]
+      - requesterComments
+
+  readOnlyRequestData:
+    readonly: true
```

Comment:
change all `readonly` fields to `readOnly` (latest version of the publisher, which is being deployed will only allow `readOnly`).